pkgs/by-name/so/sof-tools/package.nix

```diff
@@ -0,0 +1,26 @@
+{ pkgs, lib, cmake, alsa-lib }:
+
+pkgs.stdenv.mkDerivation rec {
```

Comment:
```suggestion
stdenv.stdenv.mkDerivation rec {
```

Reply:
Oh right, thanks! It's been a couple years since I contributed to nixos :)